traveler/urls.py

```diff
@@ -37,3 +39,6 @@
     path('api/v1/', include('flights.urls')),
     path('api-docs/', schema_view)
 ]
+
+if settings.DEBUG:
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
This runs when in development mode. What would happen in production?

Reply:
It will still access the same URL, just that in Debug mode it has to be specified.